dev-infrastructure/modules/keyvault/keyvault-secret-access.bicep

```diff
@@ -0,0 +1,57 @@
+@description('The name of the keyvault where access is managed')
+param keyVaultName string
+
+@description('Optional name of the secret in the keyvault where access is managed. If not provided, access will be granted to all secrets in the keyvault.')
+param secretName string = ''
+
+@description('The name of the role that will be assigned to the managed identity for the secret in KV')
+@allowed([
+  'Key Vault Secrets Officer'
+  'Key Vault Secrets User'
```

Comment:
Don't we also need Key Vaults Certificate User? What's the reason for granting the identity the officer role? 🤔

Reply:
We'll need that eventually.  

But for now, when you create a certificate, you download the secret (which includes the private half of the certificate).  So you technically don't need certificate user I believe.

Reply:
thanks for the explanation @bennerv, looks good to me!